examples/src/main/java/io/opencensus/examples/zpages/ZPagesTester.java

```diff
@@ -16,14 +16,87 @@
 
 package io.opencensus.examples.zpages;
 
+import io.opencensus.common.Scope;
+import io.opencensus.contrib.grpc.metrics.RpcMeasureConstants;
+import io.opencensus.contrib.grpc.metrics.RpcViews;
 import io.opencensus.contrib.zpages.ZPageHandlers;
+import io.opencensus.stats.MeasureMap;
+import io.opencensus.stats.Stats;
+import io.opencensus.stats.StatsRecorder;
+import io.opencensus.tags.TagValue;
+import io.opencensus.tags.Tagger;
+import io.opencensus.tags.Tags;
+import io.opencensus.trace.SpanBuilder;
+import io.opencensus.trace.Tracer;
+import io.opencensus.trace.Tracing;
+import io.opencensus.trace.samplers.Samplers;
+import java.util.Collections;
 
 /** Testing only class for the UI. */
 public class ZPagesTester {
+
   private ZPagesTester() {}
 
+  private static final Tagger tagger = Tags.getTagger();
+  private static final Tracer tracer = Tracing.getTracer();
+  private static final StatsRecorder statsRecorder = Stats.getStatsRecorder();
+
+  private static final String SPAN_NAME = "SampleSpan";
+
   /** Main method. */
   public static void main(String[] args) throws Exception {
+    Tracing.getExportComponent()
```

Comment:
Suggestion: It might be better to put this section of new code in a separate method, such as "createExampleData", to make it clear why the span and stats are being recorded.

Reply:
Done.